content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Monitoring Linux host metrics with the node exporter
```

Comment:
It's the Node exporter with a capital N

Reply:
Why not Node Exporter?

Reply:
Yeah, I would also capitalize both words, not just one.

Reply:
Done